### DISCUSSION TITLE
Disable Add-Ons and/or Scripting for all users

### DISCUSSION THREAD

**Discussion by UrLocalAdmin0298** [freeplane/freeplane#1211] [Help] [answered]
Hi everyone.
We want to establish FreePlane in our company. In terms of IT security, the use/execution of scripts and add-ons should be prevented if possible. Users should not be able to reactivate those features in the Preferences section. 
Removing the folder "C:\Program Files\Freeplane\plugins\org.freeplane.plugin.script" hides the Script-menu, but shows a warning that the script editor couldn't be found every time you start FreePlane.
Is there a config-file you can edit to disable the scripting and/or add-on feature? In best case, this file should be located in the Program Files section to prevent users from changing those settings.

**Comment by macmarrum** [comment]
You also need to disable the formula plugin, i.e. delete "C:\Program Files\Freeplane\plugins\org.freeplane.plugin.formula", and you should be good to go.

**Reply by UrLocalAdmin0298** [reply]
Thanks for the hint. 
I've deleted org.freeplane.plugin.formula and org.freeplane.plugin.script. When starting FreePlane, I got this error report: 
freeplane_version = 1.10.6; freeplane_xml_version = freeplane 1.9.13
git revision = 3e6a1230ca958fab95cb83264fd638479f278426 Dimitry Polivaev 2022-12-07T13:14:37Z
java_version = 15.0.7
missing key ScriptEditor

**Reply by dpolivaev** [reply, marked as answer]
remove line `<Entry name="ScriptEditor" />`from file mindmapmodemenu.xml

**Reply by UrLocalAdmin0298** [reply]
It works! Thank you!!
